cmd/terramate/cli/run.go

```diff
@@ -165,12 +189,16 @@ func (c *cli) runOnStacks() {
 // stacks.
 // If SIGINT is sent 3x then Terramate will send a SIGKILL to the currently
 // running process and abort the execution of all subsequent stacks.
-func (c *cli) RunAll(runStacks []ExecContext, isSuccessCode func(exitCode int) bool) error {
+func (c *cli) RunAll(
+	runs []RunContext,
+	isSuccessCode func(exitCode int) bool,
```

Comment:
the isSuccessCode seems to be the same for all callers, we could remove it as an argument and do it inline? 🤔

Reply:
For `run script` its always the same, but for `run` it can have different values, see `runOnStacks()`:
```
if c.parsedArgs.Run.CloudSyncDriftStatus {
	isSuccessExit = func(exitCode int) bool {
		return exitCode == 0 || exitCode == 2
	}
}
```